lib/reporters/dots.js

```diff
@@ -0,0 +1,74 @@
+'use strict';
+const chalk = require('chalk'),
+  _ = require('lodash'),
+  term = require('terminal-logger')('dots');
+
+term.level = 'debug'; // log everything
+chalk.enabled = true;
+chalk.level = 1;
+
+/**
+ * log simple messages
+ * @param  {string} message
+ */
+function log(message) {
+  term.status.info(message);
+}
+
+/**
+ * log each operation as it happens
+ * @param  {object} action
+ */
+function logAction(action) {
+  if (action.type === 'success') {
+    process.stderr.write(chalk.green('.'));
```

Comment:
I think it could be useful to use different symbols for these in case of people who dont have colors in terminals or have red/green colorblindness

Reply:
I notice we use ticks and crosses below, maybe just use those?

Reply:
For the dot reporter, I'm following the standard that mocha and other testing frameworks use. I agree that it might not be ideal, but I think the other reporters (specifically `pretty`) help in this regard.